python-venv/cmstools/test/cfs_sessions_rc_test/cfs/options.py

```diff
@@ -0,0 +1,73 @@
+
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+"""
+CFS options related functions
+"""
+from typing import Any
+
+from cmstools.lib.api import request
+from cmstools.lib.cfs.defs import CFS_OPTIONS_URL, CFS_DEFAULT_PAGE_SIZE
+from cmstools.test.cfs_sessions_rc_test.log import logger
+from cmstools.test.cfs_sessions_rc_test.defs import CFSRCException
+
+
+def get_cfs_options() -> dict[str, Any]:
```

Comment:
We can be more specific on the dict type, since we are returning directly from a JSON decode function.
```suggestion
from cmstools.lib.api import request
from cmstools.lib.cfs.defs import CFS_OPTIONS_URL, CFS_DEFAULT_PAGE_SIZE
from cmstools.lib.defs import JsonDict
from cmstools.test.cfs_sessions_rc_test.log import logger
from cmstools.test.cfs_sessions_rc_test.defs import CFSRCException


def get_cfs_options() -> JsonDict:
```